### PR TITLE
T9: Build Landing Page

### PR DESCRIPTION
**Pull request by fahadakhan2**
**Description:**
- Created two new components: LandingPageBody & LandingPageNavBar
- The LandingPageBody is temporary text for now
- The LandingPageNavBar only includes one navigation on it which will take the user to the Sign In page
- Changed the naming style in which the functional components are being exported

**Visual:**
Screenshot 2024-06-16 at 4 21 46 PM

### CODE REVIEW

File: frontend/src/components/LandingPage/LandingPageBody.module.css [hunk 1]
```diff
@@ -0,0 +1,6 @@
+.tempText {
```
Comment (justinpchen94) [inline frontend/src/components/LandingPage/LandingPageBody.module.css]: nit: generally css standardizes on kebab case 

if you want to go deeper, see: https://getbem.com/naming/

File: frontend/src/components/LandingPage/LandingPageNavBar.module.css [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  margin: 0;
+  overflow: hidden;
+  flex-flow: row wrap;
+  font-family: 'Poppins', sans-serif;
+  font-weight: 400;
+  font-style: normal;
+  padding: 20px;
+}
+
+.navLink {
```
Comment (justinpchen94) [inline frontend/src/components/LandingPage/LandingPageNavBar.module.css]: same wrt casing

**Review by justinpchen94 — APPROVED** [review]
pretty nice small pr, lgtm

File: frontend/src/components/LandingPage/LandingPageBody.module.css (hunk 1 above)
Comment (fahadakhan2) [reply frontend/src/components/LandingPage/LandingPageBody.module.css]: Ok will change to kebab case

File: frontend/src/components/LandingPage/LandingPageNavBar.module.css (hunk 2 above)
Comment (fahadakhan2) [reply frontend/src/components/LandingPage/LandingPageNavBar.module.css]: Done

**Pull request merged**